src/main/java/io/phasetwo/service/resource/UserResource.java

```diff
@@ -14,57 +16,62 @@
+        super(session);
+    }
 
-  ////////
-  // Users
-  ////////
+    ////////
+    // Users
+    ////////
 
-  @GET
-  @Path("/{userId}/orgs")
-  @Produces(MediaType.APPLICATION_JSON)
-  public Stream<Organization> listUserOrgs(@PathParam("userId") String userId) {
-    log.debugv("Get org memberships for %s %s", realm.getName(), userId);
+    @GET
+    @Path("/{userId}/orgs")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Stream<Organization> listUserOrgs(@PathParam("userId") String userId) {
+        log.debugv("Get org memberships for %s %s", realm.getName(), userId);
 
-    UserModel user = session.users().getUserById(realm, userId);
-    return orgs.getUserOrganizationsStream(realm, user)
-        .filter(m -> (auth.hasViewOrgs() || auth.hasOrgViewOrg(m)))
-        .map(m -> convertOrganizationModelToOrganization(m));
-  }
+        UserModel user = session.users().getUserById(realm, userId);
+        return orgs.getUserOrganizationsStream(realm, user)
+                .filter(m -> (auth.hasViewOrgs() || auth.hasOrgViewOrg(m)))
+                .map(m -> convertOrganizationModelToOrganization(m));
+    }
 
-  @GET
-  @Path("/{userId}/orgs/{orgId}/roles")
-  @Produces(MediaType.APPLICATION_JSON)
-  public Stream<OrganizationRole> listUserOrgRoles(
-      @PathParam("userId") String userId, @PathParam("orgId") String orgId) {
-    log.debugv("Get org roles for %s %s %s", realm.getName(), userId, orgId);
-
-    UserModel user = session.users().getUserById(realm, userId);
-    OrganizationModel org = orgs.getOrganizationById(realm, orgId);
-    if (auth.hasViewOrgs() || auth.hasOrgViewRoles(org)) {
-      if (org.hasMembership(user)) {
-        return org.getRolesStream()
-            .filter(r -> r.hasRole(user))
-            .map(r -> convertOrganizationRole(r));
-      } else {
-        throw new NotFoundException("User is not a member of the organization");
-      }
-    } else {
-      throw new NotAuthorizedException("Insufficient permissions");
+    @GET
+    @Path("/{userId}/orgs/{orgId}/roles")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Stream<OrganizationRole> listUserOrgRoles(
+            @PathParam("userId") String userId, @PathParam("orgId") String orgId) {
+        log.debugv("Get org roles for %s %s %s", realm.getName(), userId, orgId);
+
+        UserModel user = session.users().getUserById(realm, userId);
+        OrganizationModel org = orgs.getOrganizationById(realm, orgId);
+        if (auth.hasViewOrgs() || auth.hasOrgViewRoles(org)) {
+            if (org.hasMembership(user)) {
+                return org.getRolesStream()
+                        .filter(r -> r.hasRole(user))
+                        .map(r -> convertOrganizationRole(r));
+            } else {
+                throw new NotFoundException("User is not a member of the organization");
+            }
+        } else {
+            throw new NotAuthorizedException("Insufficient permissions");
+        }
     }
```

Comment:
This seems like just a formatting change. Can you revert?

Reply:
Done =)